lib/views/photo_booth_screen/photo_booth.dart

```diff
@@ -96,3 +112,53 @@ class PhotoBoothState extends State<PhotoBooth> {
     super.dispose();
   }
 }
+
+Widget _menuBar(BuildContext context, GoRouter router) {
```

Comment:
We should convert this function to an actual widget because of its size. There might be a built in refactor to help doing this. I suggest we move it to a `photo_booth.menu.dart` file.

The menu itself is very nice, really like it!